index.tt

```diff
@@ -6,152 +6,353 @@
 
 <link rel="alternate" type="application/rss+xml" title="RSS" href="/news-rss.xml" />
 
-<div class="nixos-org-index">
-  [%# Title for the site, used to keep the document tree rooted at h1. %]
-  <h1>NixOS</h1>
-  
-  <div class="jumbotron-two"><div>
-    <div class="intro-nix">
-      <h2>What is Nix?</h2>
-    
-      <p>
-      Nix is a powerful package manager for Linux and other Unix systems
-      that makes package management reliable and reproducible.  It
-      provides atomic upgrades and rollbacks, side-by-side installation
-      of multiple versions of a package, multi-user package management
-      and easy setup of build environments.
-      </p>
-    
-      <div class="learn-more">
-        <a class="btn btn-success" href="[%root%]nix/">
-          <i class="fa fa-chevron-right"></i> Learn more about Nix
-        </a>
+<div class="row-fluid">
+
+  <div class="span6">
+    <h1>Reproducible builds and deployments.</h1>
+    <p class="lead">
+      NixOS package manager can be used independently on any POSIX compliant
+      operating system to reliably share your development and build
+      environments across machines.
+    </p>
+    <p class="lead">
+      NixOS, an independently developed GNU/Linux distribution, aims to improve
+      the state of the art in system configuration management. The entire
+      operating system, including the kernel, applications, system packages and
+      configuration files, are built from a single location.
+    </p>
+    <!-- <a class="btn btn-large" href="[% root %]download.html">Why NixOS</a> -->
+    <a class="btn btn-large btn-success" href="[% root %]learn.html">Get started</a>
+    <a class="btn btn-large" href="[% root %]download.html">Download</a>
+    <a class="" href="https://github.com">View on GitHub</a>
```

Comment:
Do we need "View on GitHub"? In any case it currently links to the GitHub front page.

Reply:
in my opinion no. it's the last place where you go, when you start contributing as a Nix/NixOS user

such a link belongs in the footer next to social media links

Reply:
removed